webapp/channels/src/components/admin_console/permission_schemes_settings/permissions_scheme_summary/permissions_scheme_summary.tsx

```diff
@@ -148,17 +146,13 @@ export default class PermissionsSchemeSummary extends React.PureComponent<Props
         let extraTeams = null;
         if (teams.length > MAX_TEAMS_PER_SCHEME_SUMMARY) {
             extraTeams = (
-                <OverlayTrigger
-                    delayShow={Constants.OVERLAY_TIME_DELAY}
-                    placement='bottom'
-                    overlay={
-                        <Tooltip
-                            className='team-scheme-extra-teams-overlay'
```

Comment:
can you screenshot this because we are loosing some styling with not including `team-scheme-extra-teams-overlay`

Reply:
I have restored the styling but unfortunately I am not able to provide a screenshot for this file. I used the tooltip migration I did on `restricted_indicator` file as a reference

Reply:
i was able to find out which component this tooltip belongs
![CleanShot 2024-08-16 at 22 55 38](https://github.com/user-attachments/assets/d96fb1c1-8443-460b-812f-581e94179a0b)

Reply:
Lets add it as follows
```
title={this.props?.teams?.slice(MAX_TEAMS_PER_SCHEME_SUMMARY).map((team) => team.display_name).join(', ') ?? ''}
```